src/averageAges.js

```diff
@@ -15,12 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let age = 0;
+
+  const m = people.filter(x => x.sex === 'm');
+  const a = m.filter(x => Math.ceil(x.died / 100) === century);
```

Comment:
```suggestion
  const m = people.filter(x => x.sex === 'm');
  const a = m.filter(x => Math.ceil(x.died / 100) === century);
```
Дуже погані назви m та a. Взагалі не зрозуміло, що б вони мали означати